rules/xcodeproj.bzl

```diff
@@ -5,7 +5,9 @@ load("@build_bazel_rules_apple//apple/internal:platform_support.bzl", "platform_
 load("@build_bazel_rules_swift//swift:swift.bzl", "SwiftInfo")
 load("@bazel_skylib//lib:paths.bzl", "paths")
 load("//rules:hmap.bzl", "HeaderMapInfo")
+load("//rules/framework:vfs_overlay.bzl", "VFSOverlayInfo", VFS_OVERLAY_FRAMEWORK_SEARCH_PATH = "FRAMEWORK_SEARCH_PATH")
```

Comment:
never seen this syntax before ` VFS_OVERLAY_FRAMEWORK_SEARCH_PATH = "FRAMEWORK_SEARCH_PATH"` is this just to declare a constant? 🙏

Reply:
Yeah this is a starlarkism  to import a symbol under a different name, it just makes for better semantic usage from file to file. I can dig up the docs on the `load` for this if you'd want to read more about it

Kind of like pythons `import x from y as f` notion